application/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+logging.level.root=INFO
```

Comment:
In a spring application would it make sense to move the properties defined in the Main class to this properties file?